task/models.py

```diff
@@ -20,10 +20,11 @@ class Task(models.Model):
     def __str__(self) -> str:
         return self.title
     
+    
 class Rating(models.Model):
-    manager = models.ForeignKey(User, on_delete=models.CASCADE)
     title = models.ForeignKey(Task, on_delete=models.CASCADE)
```

Comment:
use a proper naming convention, why "title" for "task" ForeignKey ?

Reply:
Only the reporter (Admin/Manager) can review the task once so no need to create a separate table for review.